.circleci/config.yml

```diff
@@ -292,3 +300,52 @@ jobs:
             - target/debug/build
             - target/debug/deps
           key: cargocache-v2-lint-rust:1.44.1-{{ checksum "Cargo.lock" }}
+
+  # This job roughly follows the instructions from https://circleci.com/blog/publishing-to-github-releases-via-circleci/
+  build_and_upload_contracts:
+    docker:
+      # Image from https://github.com/cibuilds/github, based on alpine
+      - image: cibuilds/github:0.13
+    steps:
+      - run:
+          name: Install Docker client
+          command: apk add docker-cli
+      - setup_remote_docker
+      - checkout
+      - run:
+          name: Create atrifacts directory
+          command: mkdir ./artifacts
+      - run:
+          # We cannot mount local folders, see https://circleci.com/docs/2.0/building-docker-images/#mounting-folders
+          name: Prepare volume with source code
+          command: |
+            # create a dummy container which will hold a volume with config
+            docker create -v /code --name with_code alpine /bin/true
+            # copy a config file into this volume
+            docker cp ./contracts with_code:/code
+            docker cp ./packages with_code:/code
+      - run:
+          name: Build development contracts
+          command: |
+            for contract_dir in ./contracts/*/; do
```

Comment:
Okay, this needs to be updated for 0.10.0 rust-optimizer, but the rest looks quite straight-forward.

Thanks for this PR